scripts/compose.py

```diff
@@ -388,16 +395,23 @@ def add_arguments(cls, parser):
             default='elasticsearch',
             help='apm-server output'
         )
+        parser.add_argument(
+            "--no-apm-server-dashboards",
+            action="store_false",
+            dest="apm_server_dashboards",
+            help="apm-server output",
```

Comment:
Maybe some more useful help text, here and/or in readme, also mentioning that it doesn't take effect kibana is disabled
Particularly, if we are pointing non apm devs to this repo...

Reply:
nice catch, fixing